packages/playground/src/components/pull-request/sections/pull-request-comment-section.tsx

```diff
@@ -19,7 +19,7 @@ const PullRequestCommentSection = ({ commentsInfo, handleAction }: PullRequestMe
                 commentsInfo.status === 'success' ? (
                   <Icon name="success" size={16} />
                 ) : (
-                  <WarningTriangleSolid className="text-destructive mt-1" />
+                  <WarningTriangleSolid className="text-destructive" />
```

Comment:
should we use the icon here

Reply:
Yes, you mean the <Icon name="" /> right? If so, yes you would want to replace that.